WordPress/gutenberg
# Revert: Block.json: Refactor and stabilize selectors API

## What?
Reverts https://github.com/WordPress/gutenberg/pull/46496 because I merged before it was ready.

## Why?
There are a few issues that arose as a result of this. Lets address the issues in another PR rather than risk shipping Gutenberg in a broken state.

## How?
This had to be a manual revert because the revert button didn't work.

## Testing Instructions
- Read the code
- Check the testing instructions from https://github.com/WordPress/gutenberg/pull/46496
